lib/components/app/app-menu.js

```diff
@@ -41,9 +47,92 @@ class AppMenu extends Component {
     window.location.href = startOverUrl
   }
 
+  _expandedPane = () => {
+    const { isPaneOpenLeft } = this.state
+    this.setState({ isPaneOpenLeft: !isPaneOpenLeft })
+  }
+
+  _expandSubmenu = () => {
+    const { isSubmenuExpanded } = this.state
+    this.setState({ isSubmenuExpanded: !isSubmenuExpanded })
+  }
+
+  _addExtraMenuItems = (menuItems) => {
+    const AddIconAndLabel = ({iconType, iconUrl, label}) => {
+      return (
+        iconUrl
+          ? <span>
+            <img alt={`icon for ${label} Menu Item`} src={iconUrl || 'bus'} />
+            {label}
+          </span>
+          : <span><Icon name={iconType || 'car'} />{label}</span>
```

Comment:
```suggestion
          : <span><Icon name={iconType || 'external-link-square'} />{label}</span>
```

I took a look at font awesome and this was the best default icon I could find. What do you think?

Reply:
@miles-grant-ibigroup what happens if it is not a external link? Other ideas could be:

- [Arrow signs](https://fontawesome.com/v5.15/icons/map-signs?style=solid)
- [Directions](https://fontawesome.com/v5.15/icons/directions?style=solid)
- [Atlas Book](https://fontawesome.com/v5.15/icons/atlas?style=solid)

Reply:
I assumed that the default would always be an external link but I like the arrow signs!